### ISSUE TITLE
clue复现运行时问题

### ISSUE THREAD

**Issue by jack18501318588** [IDEA-CCNL/Fengshenbang-LM#277]
报警：/home/aipf/.local/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/data_connector.py:224: PossibleUserWarning: The dataloader, train_dataloader, does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` (try 56 which is the number of cpus on this machine) in the `DataLoader` init to improve performance.
命令行：TASK="tnews"
Testdata="test1.1.json"
!python /home/aipf/work/Fengshenbang-LM/fengshen/examples/clue1.1/solution/clue_unimc.py --pretrained_model_path IDEA-CCNL/Erlangshen-UniMC-RoBERTa-110M-Chinese --output_path /home/aipf/work/Fengshenbang-LM/fengshen/examples/clue1.1/predict/$TASK_predict.json --train --data_dir /home/aipf/work/Fengshenbang-LM/fengshen/examples/clue1.1/data/$TASK --train_data train.json --valid_data dev.json --test_data  $Testdata --batchsize 16 --max_length 512 --learning_rate 0.000002 --weight_decay 0.1 --warmup 0.06 --monitor val_acc --save_top_k 3 --mode max --every_n_train_steps 100 --save_ckpt_path  /home/aipf/work/Fengshenbang-LM/fengshen/examples/clue1.1/checkpoint/$TASK_checkpoint --filename model-{epoch:02d}-{val_acc:.4f} --max_epochs 3 --gpus 0,1, --check_val_every_n_epoch 1 --val_check_interval 100 --gradient_clip_val 0.25 --default_root_dir /home/aipf/work/Fengshenbang-LM/fengshen/examples/clue1.1/log

**Comment by suolyer** [comment]
这里没有报错信息，你能提供一下吗
